gslib/commands/signurl.py

```diff
@@ -445,10 +458,15 @@ def _ParseAndCheckSubOpts(self):
           'url arguments if the --use-service-account flag is missing. '
           'Run `gsutil help signurl` for more info')
 
-    return method, delta, content_type, passwd, region, use_service_account
+    if use_service_account and billing_project:
+      raise CommandException(
+          'Specifying both the -b and -u options together is invalid.')
```

Comment:
I think this is ambiguous: within the signurl command, there's also an unrelated -u flag and I don't think that one would be incompatible with -b, right?

Reply:
That's exactly what I meant here... the `-u` option is actually using a totally different API where specifying a billing project would have no effect.